app/src/main/java/com/example/geoquiz/QuizActivity.kt

```diff
@@ -131,7 +136,15 @@ class QuizActivity : AppCompatActivity() {
         }
 
         cheatButton.setOnClickListener { view ->
-            openCheatActivity(view)
+            if (quizViewModel.numberOfCheats > 0) {
+                openCheatActivity(view)
+                quizViewModel.numberOfCheats--
+            } else {
+                cheatButton.isClickable = false
```

Comment:
If we make the button unclickable, the user will think that the app has frozen or is not working.
Let's instead display a toast explaining that cheats have been consumed